src/build.jl

```diff
@@ -14,6 +14,10 @@ function swap_our_param_with_jump_param(expr)
     end
 end
 
+"""This function takes an expression tree and replaces all instances of
+`JuMP.NLParameter` with the corresponding `ParameterRef`.
+"""
+
```

Comment:
If you remove the empty line between this string and the function itself, then it will associate the string as the documentation with the function :) See [here](https://docs.julialang.org/en/v1/manual/documentation/) how that works.

Reply:
Ok, yes, got it. Thanks.

Reply:
Ok, yes, got it. Thanks.